lib/ansible/modules/packaging/language/pip.py

```diff
@@ -674,7 +682,8 @@ def main():
 
             changed = False
             if name:
-                pkg_list = [p for p in out.split('\n') if not p.startswith('You are using') and not p.startswith('You should consider') and p]
+                pkg_list = [p for p in out.split('\n')
```

Comment:
It looks like this hasn't been changed. Please don't do any unrelated changes. It's makes it hard do review PRs.

Reply:
I'm pretty sure I did it because it was reporting a long line. I must have changed this line before, so than it was long and I never returned it back. Anyway since this is where the pkg_list is initialized, this will have to be split into two lines, because PEP is failing with long line.